Pipfile

```diff
@@ -7,10 +7,11 @@ verify_ssl = true
 coverage = "~=4.5"
 requests = "~=2.23"
 unittest-xml-reporting = "~=2.5"
+#integration-adaptors-common = {editable = true, path = "./../integration-adaptor-common"}
```

Comment:
remove commented line?

Reply:
I think that it would be left here commented, it is for using local  common library. It would be easier just to uncomment this instead of writing it once and once again

Reply:
exactly as @Alicja-Kainos wrote